pkg/alertmanager/amcfg.go

```diff
@@ -589,7 +589,7 @@ func (cg *configGenerator) convertOpsgenieConfig(ctx context.Context, in monitor
 
 	var responders []opsgenieResponder
 	if l := len(in.Responders); l > 0 {
-		responders = make([]opsgenieResponder, l)
+		responders = []opsgenieResponder{}
```

Comment:
I think the original intent was:

```suggestion
		responders = make([]opsgenieResponder, 0, l)
```